packages/config/src/Config.ts

```diff
@@ -50,7 +51,7 @@ export class Config {
     private _vault: IConfigVault;
     private _secure: IConfigSecure;
 
-    private constructor() { }
+    private constructor(public opts?: IConfigOpts) { }
```

Comment:
Curious why `opts` was added here, it seems unused since constructor is empty?

Reply:
it's empty but still has an assignment built in.  it's used on line 375 in packages/cmd/src/CommandProcessor.ts